.changeset/sharp-stingrays-sip.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cambly/syntax-core": major
```

Comment:
this should be a minor bump

Reply:
to clarify: it's a minor bump because upgrading this version of Syntax doesn't break any of our usages of Syntax going from our previous version

Reply:
ah ic thanks for letting me know